packages/components/src/ListingSummary/ListingSummaryRejected.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from "react";
+import { EthAddress } from "@joincivil/core";
+import { buttonSizes, InvertedButton } from "../Button";
+import { ChallengeResults, ChallengeResultsProps } from "../ChallengeResultsChart";
+import {
+  StyledListingSummaryContainer,
+  StyledListingSummaryTop,
+  StyledListingSummarySection,
+  StyledListingSummaryNewsroomName,
+  ChallengeResultsContain,
+  NewsroomIcon,
+} from "./styledComponents";
+
+export interface ListingSummaryRejectedComponentProps {
+  address?: EthAddress;
+  name?: string;
+  owners?: EthAddress[];
```

Comment:
I think we can get rid of `owners`

Reply:
And probably `description` as well, since the Challenge Results replace the description, right?